readme.md

```diff
@@ -178,6 +178,12 @@ Type: `Object` `Array` `number` `string` `boolean` `null`
 
 JSON body. The `Content-Type` header will be set to `application/json` if it's not defined.
 
+###### userData
+
+Type: `unknown`
+
+User data. In the opposite to other options, `userData` is not enumerable.
```

Comment:
As a user reading this description, it's not clear what it's for or how it's useful to me as a user. I think we need to describe what they could use it for. Would also be great with a simple code example.